src/it/MCHECKSTYLE-137/checkstyle.xml

```diff
@@ -20,8 +20,6 @@ under the License.
 <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
```

Comment:
it would be good if you can update all xmls to have new DOCTYPE that reference to secure website.
Example: https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L2
There should be no `Puppy Crawl`, no `http://`.

it is part of security issue - https://github.com/checkstyle/checkstyle/issues/6478  and https://github.com/checkstyle/checkstyle/issues/6474

Reply:
This is a test resource

Reply:
@eolivelli , please address my comment, to avoid security issues in plugin.

Reply:
@romani this file is a test resource, it is an integration test.
It would be better to create a specific JIRA and track the work to do.
I will be very happy to fix the problem you are reporting or to help your provide a patch.
This PR has already been merged, we should start a new one

Reply:
@eolivelli The security issue is in the parser that is used to parse this file. It leaves any code that parses this file vulnerable to XXE via a MITM.
It doesn't matter that it's a "test resource", it still leaves the build vulnerable to this attack vector.

Reply:
@JLLeitschuh @romani I got your point.
Do you have time to create a JIRA and send a PR?
I guess we have a lot of files like this one.

This patch has already been merged

Reply:
@eolivelli I don't, sorry. I don't even use this library.
I'm just the person that originally reported the vulnerability to the Checkstyle team.

Reply:
I see

Reply:
I will create the ticket.

Reply:
Here is the new ticket to provide any patches for: https://issues.apache.org/jira/browse/MCHECKSTYLE-375

Reply:
Great. If anyone wants to pick it up....I can wait for it before cutting the 3.1.0 release.
I am currently waiting for another patch se the is no hurry.

Reply:
@eolivelli , please share link to patch that you are waiting

Reply:
@romani this one https://github.com/apache/maven-checkstyle-plugin/pull/5